recipes/zetcd/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set import_path = "github.com/etcd-io/zetcd" %}
+{% set version = "0.0.5" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace('etcd-io',
+                                              'coreos').replace("/", "\\" if win else "/") %}
+
+package:
+  name: {{ name|lower }}
+  version: v{{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    git_url: https://{{ import_path }}
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v
```

Comment:
Use `go install` instead of `go build` so that the resulting binaries get automatically installed into PREFIX/bin. This is consistent for Windows and Linux.

Reply:
duly noted on this one